prime-router/metadata/schemas/NM/nm-covid-19.schema

```diff
@@ -4,7 +4,7 @@ description: NM COVID-19
 referenceUrl:
 topic: covid-19
 trackingElement: message_id
-basedOn: covid-19
+extends: covid-19
 elements:
```

Comment:
Changed the HL7 schema to extend from COVID-19 for local testing to make sure that all the fields from the base schema come in. Was causing some weird results on my end when a few fields in `covid-19` didn't show up locally but did show up when I was running it in Docker.